packages/react-components/react-storybook-addon/src/decorators/withAriaLive.tsx

```diff
@@ -2,16 +2,22 @@ import * as React from 'react';
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32011&commitId=31ef8656c3ae1317fcfc9985e12a1d4f1696e49d&env=prod&clientType=FLUENTUI)

_Diffs exceeded the limit allowed for comments. Please use the report link above to check your diffs._


<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>